docs/advanced_reference/commands/autonomy_fetch.md

```diff
@@ -104,3 +110,105 @@ Fetch the agent service with the token ID `123` on Gnosis chain:
 ```bash
 autonomy fetch 123 --use-gnosis
 ```
+
+Fetch the agent service with the token ID `123` using on-chain token ID:
+```bash
+autonomy fetch 42 --service --chain ethereum
+```
+
+### Viewing Remote Registry Files
+
+Before fetching a service, you can inspect its contents in the remote registry:
+
+1. View service metadata:
+```bash
+# Using package ID
+autonomy packages info valory/hello_world:0.1.0 --remote
+
+# Using token ID
+autonomy packages info 42 --chain ethereum
+```
+
+2. List available versions:
+```bash
+autonomy packages list --remote | grep hello_world
+```
+
+3. Inspect service configuration:
+```bash
+# Download and view service.yaml without fetching entire package
+curl -L https://gateway.autonolas.tech/ipfs/<hash>/service.yaml
+```
+
+4. Browse IPFS contents:
+If the service is stored on IPFS, you can browse its contents through:
+- IPFS gateway: `https://gateway.autonolas.tech/ipfs/<hash>/`
+- Local IPFS node (if running): `http://localhost:8080/ipfs/<hash>/`
+
+### Viewing Service Files
+
+After fetching a service, its files are stored in your local packages directory. Here's how to view them:
+
+#### Service Directory Structure
+
+The service files are organized in the following structure:
+```
+packages/
+└── valory/
+    └── hello_world/
+        ├── service.yaml       # Service configuration
+        ├── contracts/         # Smart contracts
+        ├── protocols/         # Communication protocols
+        ├── skills/           # Service skills
+        └── vendor/           # Dependencies
+```
```

Comment:
This is not correct. Service folder only contains service.yaml and README.md. When a deployment is built, it is done on the abci_build folder, as a docker compose/kubernetes deployment. That folder contains the deployment, logs, and other config data. The agent packages are not downloaded on the vendor folder; the deployment will use the Docker image published on DockerHub.